scripts/generate_artifacts.jl

```diff
@@ -0,0 +1,59 @@
+
+using ACE1pack, Pkg
+using Pkg.Artifacts
+
+artifacts_toml = joinpath(pathof(ACE1pack)[1:end-16], "Artifacts.toml")
+
+function generate_artifact(label, zipname, url)
+    data_hash = artifact_hash(label, artifacts_toml)
+
+    if data_hash == nothing || !artifact_exists(data_hash)
+        tarfile = download(url, @__DIR__() * "/" * zipname)
+        hash_ = create_artifact() do artifact_dir
+            cp(tarfile, joinpath(artifact_dir, zipname))
+        end
+        tarball_hash = archive_artifact(hash_, joinpath(tarfile))
+        bind_artifact!(artifacts_toml, label, hash_,
+                    download_info = [ (url, tarball_hash) ], lazy=true, force=true)
+    end
+end
+
+
+label = "TiAl_tiny_dataset"
+zipname = "TiAl_tiny.zip"
+url = "https://github.com/gelzinyte/test_julia_artifacts/blob/main/$(zipname)?raw=true"
```

Comment:
will update once/if https://github.com/ACEsuit/ACEData/pull/1 is merged

Reply:
shall I go ahead and merge ACEData#1?

Reply:
actually, let me first update the yaml file and structure of basis_params

Reply:
ACEData#1 is up to date with this PR. However, it is probably easier to hold on from merging it until this is ready to be merged, in case addressing any other comments would require changing some of the files.